pkg/daemon/ceph/mon/daemon.go

```diff
@@ -128,6 +128,7 @@ func startMon(context *clusterd.Context, config *Config, confFilePath, monDataDi
 		"--mkfs",
 		monNameArg,
 		fmt.Sprintf("--cluster=%s", config.Cluster.Name),
+		fmt.Sprintf("--log-stderr-prefix=debug "),
```

Comment:
If that is `stderr` wouldn't the prefix `error` be more fitting?

Reply:
this is making the debug log (what normally goes to e.g., /var/log/ceph/ceph-mon.foo.log) to go stderr.  The goal is to disambiguate this from the cluster logs (cluster, audit, etc) that normally go to /var/log/ceph/ceph.log or ceph.audit.log